test/transport_web/solutions/open_data_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule TransportWeb.OpenDataTest do
+  @moduledoc """
```

Comment:
Here you can copy-paste the entire story, as we'll probably add successive tests here.

Reply:
done